llo/json_report_codec_test.go

```diff
@@ -16,6 +25,257 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func Fuzz_JSONCodec_Decode_Unpack(f *testing.F) {
```

Comment:
IIUC the first underscore should not be included:
```suggestion
func FuzzJSONCodec_Decode_Unpack(f *testing.F) {
```